src/main/webapp/scripts/tests/view-test.js

```diff
@@ -0,0 +1,9 @@
+QUnit.test('createHeader', function(assert) {
+  createHeader();
+  assert.dom('h3').hasText(TITLE_TEXT);
+});
+
+QUnit.test('createMenu', function(assert) {
+  createMenu();
+  assert.dom('i').exists({ count: 3});
+});
```

Comment:
It is not obvious that the 'h3' and 'i' are for the header and menu. Could you instead add id attributes and query by those here instead? Easier to read if it said "assert that the header has TITLE_TEXT" and "assert the menu has 3 items".

Reply:
changed it so that it queries by the bootstrap classes that the tags already had (navbar-brand and nav-item) which I think are descriptive enough to not have to add another class (assert that the navigation bar's brand is TITLE_TEXT and assert there are 3 navigation items)